subsys/bluetooth/controller/ll_sw/ull_llcp_features.h

```diff
@@ -132,22 +132,22 @@ static inline bool feature_cte_req(struct ll_conn *conn)
  */
 static inline bool feature_smi_rx(struct ll_conn *conn)
 {
-	return LL_FEAT_BIT_SMI_RX;
+	return LL_FEAT_BIT_SMI_RX != 0;
 }
```

Comment:
`conn` unused?

Reply:
yeap

Reply:
No, will mark arg as unused, so as to keep header identical with other similars

Reply:
I'll let this be up to you guys working in the controller, but at least I do not see the point of keeping the argument if it is unused, unless it's used with a specific callback function definition :)

Reply:
Agree - will remove